src/commands/new-proj/new-repo.ts

```diff
@@ -20,15 +15,15 @@ async function action(args) {
   let repoName : string = args.name;
   let description : string = args.options.description;
   const template = args.options.template;
+  const { HOME } = process.env;
+  const defaultPath = `${HOME}/.meta-proj-cli/projects`;
```

Comment:
shouldnt we just make this a feature in path-utils at this point

Reply:
that is left for another issue